nncf/onnx/graph/model_transformer.py

```diff
@@ -68,166 +105,100 @@ def transform(self, transformation_layout: TransformationLayout) -> onnx.ModelPr
                 model_extraction_transformation = transformation
             elif isinstance(transformation, ONNXQDQNodeRemovingCommand):
                 qdq_node_removing_transformations.append(transformation)
-
+        # Inplace transformations, using deepcopy of model
         if quantizer_insert_transformations:
-            self._apply_quantizer_insertion_transformations(quantizer_insert_transformations)
-        if output_insert_transformations:
-            self._apply_output_insertion_transformations(output_insert_transformations)
-        if bias_correction_transformations:
-            self._apply_bias_correction_transformations(bias_correction_transformations)
-        if model_extraction_transformation:
-            self._model = self._apply_model_extraction_transformation(model_extraction_transformation)
-        if qdq_node_removing_transformations:
-            self._apply_qdq_node_removing_transformation(qdq_node_removing_transformations)
-
-        return self._model
-
-    def _apply_output_insertion_transformations(self, transformations: List[ONNXOutputInsertionCommand]) -> None:
+            model = self._apply_quantizer_insertion_transformations(quantizer_insert_transformations)
+        elif bias_correction_transformations:
```

Comment:
it is not expected behavior that some commands from the transformation layout will not be applied.

Reply:
Please, take a look at the update `transformI()` method.
@alexsu52 @andrey-churkin @KodiaqQ